tensorflow/examples/udacity/1_notmnist.ipynb

```diff
@@ -191,7 +191,8 @@
         "  tar.extractall()\n",
         "  tar.close()\n",
         "  data_folders = [\n",
-        "    os.path.join(root, d) for d in sorted(os.listdir(root)) if d != '.DS_Store']\n",
+        "    os.path.join(root, d) for d in sorted(os.listdir(root))\n",
```

Comment:
Pushed. This is no longer needed. Happy to merge the .gitignore change when you revert this.